tests/integ/test_cte.py

```diff
@@ -227,5 +227,4 @@ def test_table_update_delete_merge(session):
 def test_explain(session):
     df = session.create_dataframe([[1, 2], [3, 4]], schema=["a", "b"])
     explain_string = df.union_all(df)._explain_string()
-    assert "WithReference" in explain_string
```

Comment:
Is this test being run currently?

Reply:
Could you please also fill in the description?

Reply:
Yes, done